.travis.yml

```diff
@@ -10,6 +10,12 @@ python:
 - '3.9'
 services:
 - docker
+jobs:
+  include:
+  - language: python
+    python: '3.9'
```

Comment:
Are we using python 3.9 because it's the latest python version we support?

Reply:
correct